docs/source/models/intervention_models/intrapartum/azithromycin_intervention.rst

```diff
@@ -0,0 +1,265 @@
+        CEmONC_w [label="p_sepsis_with_azithromycin"]
+
+        facility -> home  [label = "home birth"]
+        facility -> BEmONC  [label = "BEmONC"]
+        facility -> CEmONC  [label = "CEmONC"]
+
+        BEmONC -> BEmONC_w  [label = "available"]
+        BEmONC -> BEmONC_wo  [label = "unavailable"]
+
+        CEmONC -> CEmONC_w  [label = "available"]
+        CEmONC -> CEmONC_wo  [label = "unavailable"]
+    }
+
+.. math::
+    \begin{align*}
+        p(\text{sepsis}) 
+        &= \sum_{\text{paths without azithromycin}} p(\text{path})\cdot p(\text{sepsis}|\text{no azithromycin})\\
+        &+ \sum_{\text{paths with azithromycin}} p(\text{path})\cdot p(\text{sepsis}|\text{azithromycin})\\[.1in]
+        p(\text{sepsis}|\text{no azithromycin}) &= \text{RR}_\text{no azithromycin} \cdot p(\text{sepsis}|\text{azithromycin})
+    \end{align*}
+
+where :math:`p(\text{sepsis})` is the probability of contracting sepsis in the general population, and :math:`p(\text{sepsis}|\text{azithromycin})` and
+ :math:`p(\text{sepsis}|\text{no azithromycin})` are the probability of contracting sepsis in settings with and without access to azithromycin.  For each 
+ path through the decision tree, :math:`p(\text{path})` is the probability of that path; for example the path that includes the edges labeled BEmONC and 
+ unavailable occurs with probability that the birth is in a BEmONC facility times the probability that the facility has azithromycin available.
+
+When we fill in the location-specific values for delivery facility rates, azithromycin coverage, relative risk of sepsis incidence with azithromycin access, 
+and sepsis incidence probability (which is also age-specific), this becomes a system of two linear equations with two unknowns (:math:`p(\text{sepsis}|\text{azithromycin})` 
+and :math:`p(\text{sepsis}|\text{no azithromycin})`), which we can solve analytically using the same approach as in the :ref:`cpap calibration <cpap_calibration>`.
+
+**Alternative PAF Derivation**: An alternative, and possibly simpler derivation of the PAF that will calibrate this model comes from the observation that
+:math:`\text{PAF} = 1 - \frac{1}{\mathbb{E}(\text{RR})}`.  If we define 
+
+.. math::
+
+   p(\text{no azithromycin}) = \sum_{\text{paths without azithromycin}} p(\text{path}),
+
+then can use this to expand the identity
+
+.. math::
+
+   \text{PAF}_\text{no azithromycin} = 1 - \frac{1}{\mathbb{E}(\text{RR})}.
+
+Since our risk exposure has two categories,
+
+.. math::
+
+   \mathbb{E}(\text{RR}) = p(\text{no azithromycin}) \cdot \text{RR}_\text{no azithromycin} + (1 - p(\text{no azithromycin})) \cdot 1.
+
+
+
+Scenarios
+---------
+
+.. todo::
+
+  Describe our general approach to scenarios, for example set coverage to different levels in different types of health facilities; then the specific values for specific scenarios will be specified in the :ref:`MNCNH Portfolio model <2024_concept_model_vivarium_mncnh_portfolio>`.
+
```

Comment:
I think we should delete this section because we now have a specific spot for this on the concept model doc that you can directly add scenarios to.